### PR TITLE
feat(chore): add contributing docs

### PR DESCRIPTION
**Pull request by hoang-dao**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -109,6 +109,9 @@ dist
 # TernJS port file
 .tern-port
 
+# VSCode configs
+.vscode
```
Comment (manh-vv) [inline .gitignore:113]: @hoang-dao .vscode is added to the git on the purpose of suggesting configuration of vs. There is no mistake here.

**Comment by manh-vv** [comment]
I pass reviewing the content of CONTRIBUTING.md to @flowersinthesand and @quocle108.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+2. [How To Develop Features](#how-to-develop)
+3. [How To Report Bugs](#how-to-report-bugs)
+
+## Getting Involved
+
+We're always looking for help [implementing features](#how-to-develop), [identifying bugs](#how-to-report-bugs), writing test cases, and improving documentation. If you find any issues, please don't hesitate to discuss with us, any ideas are helps for us.
+
+There're some related guides for you to check before contributing to our project:
+- [High-level Architecture](https://github.com/lecle/aloxide/wiki/High-level-Architecture)
+- [Commit convention]()
```
Comment (flowersinthesand) [inline CONTRIBUTING.md:13]: The link is missing?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### Adding new ones
+
+It would be great to hear new ideas, features contribute to Aloxide, if you have any, don't hesitate to let us know about that.
+
+## How To Report Bugs
+
+### Make sure it is an Aloxide bug
+
+Most bugs reported maybe actually bugs in user code, not in Aloxide code. Keep in mind that just because your code throws an error inside of Aloxide, this does *not* mean the bug is a Aloxide bug.
```
Comment (flowersinthesand) [inline CONTRIBUTING.md:29]: It's likely to be true. However, users may want to confirm whether it's a bug or not. Let's guide users to post questions to our groups, https://groups.google.com/g/aloxide, if they're not sure where bug is located or just have some ideas to improve Aloxide.

File: .gitignore (hunk 1 above)
Comment (hoang-dao) [reply .gitignore:113]: I think we shouldn't commit this settings to our project, VSCode settings is personalized configs, it's based on the style that the developer want to work with, forcing anyone to use a specific configs is not a good idea.

File: CONTRIBUTING.md (hunk 2 above)
Comment (hoang-dao) [reply CONTRIBUTING.md:13]: I was adding commit convention to the wiki but remembering that we don't discuss about this convention yet, I'm going to remove it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (hoang-dao) [reply CONTRIBUTING.md:29]: @flowersinthesand Please check if this is okay:

> Any contributions are precious to us, if you have any troubles or just have some ideas to improve Aloxide, please don't hesitate to send us your concerns at our groups https://groups.google.com/g/aloxide, we're happy and willing to help checking at anytime when we're available.

File: .gitignore (hunk 1 above)
Comment (flowersinthesand) [reply .gitignore:113]: I basically agree with @hoang-dao as members can use `.git/info/exclude` to hide their personal settings but if the team decided to use VSCode or whatever as an official IDE for some convention I think it's fine to include it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (flowersinthesand) [reply CONTRIBUTING.md:29]: LGTM

**Comment by hoang-dao** [comment]
@manh-vv @flowersinthesand please help me check again

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# Contributing to Aloxide
+
+1. [Getting Involved](#getting-involved)
+2. [How To Develop Features](#how-to-develop)
+3. [How To Report Bugs](#how-to-report-bugs)
+
+## Getting Involved
+
+We're always looking for help [implementing features](#how-to-develop), [identifying bugs](#how-to-report-bugs), writing test cases, and improving documentation. If you find any issues, please don't hesitate to discuss with us, any ideas are helps for us.
```
Comment (tiendq) [inline CONTRIBUTING.md:9]: finding bugs

any ideas are helpful for us

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+There're some related guides for you to check before contributing to our project:
+- [High-level Architecture](https://github.com/lecle/aloxide/wiki/High-level-Architecture)
+
+## How To Develop features
```
Comment (tiendq) [inline CONTRIBUTING.md:14]: Features

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# Contributing to Aloxide
+
+1. [Getting Involved](#getting-involved)
+2. [How To Develop Features](#how-to-develop)
```
Comment (tiendq) [inline CONTRIBUTING.md:4]: Add, create, or suggest etc. is more specific. Develop is a generic word.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+## How To Report Bugs
+
+### Send us your concerns
+
+Any contributions are precious to us, if you have any troubles or just have some ideas to improve Aloxide, please don't hesitate to send us your concerns at our groups https://groups.google.com/g/aloxide, we're happy and willing to help checking at anytime when we're available.
+
+### Try the latest version of Aloxide
+
+Bugs in old versions of Aloxide may have already been fixed. In order to avoid reporting known issues, make sure you are always testing against the latest build.
+
```
Comment (tiendq) [inline CONTRIBUTING.md:33]: Please also do a quick search in the [issues](link here) page to see if your issue has been raised by others.

**Review by tiendq — CHANGES_REQUESTED** [review]
Please take a look at my comments.

File: CONTRIBUTING.md (hunk 6 above)
Comment (hoang-dao) [reply CONTRIBUTING.md:4]: I split it into smaller detailed sections in `How To Develop Features` so I think we don't need to make it so clear here.

**Comment by hoang-dao** [comment]
@tiendq Thanks for your review, I updated the docs, please help me check again

**Comment by tiendq** [comment]
> @tiendq Thanks for your review, I updated the docs, please help me check again

LGTM.

**Pull request merged**